docs/can-observe.md

```diff
@@ -25,33 +26,37 @@ dog.name = 'Wilbur';
 
 ## Use
 
-Using __can-observe__ allows you to create observable objects for which any property added can be observed. It is like [can-map] but without the required [can-map.prototype.attr attr] method. This makes can-observe ideal for use-cases where the data that needs to be observed might be dynamic, or where the more rigid approach of [can-define] is unneeded.
+Using `can-observe` allows you to create observable objects with any property added can be observed, including nested objects. It is like [can-map] but without the required [can-map.prototype.attr attr] method. This makes `can-observe` ideal for use-cases where the data may be dynamic, or where the more rigid approach of [can-define] is unneeded.
```

Comment:
> objects with any property added can be observed

Doesn't read right.  `for which` or `were` makes more sense.  I also think "any property" being observed isn't really the "selling point".  These are Objects that look and act just like Objects and Arrays.  They just happen to be observable.

We should remove references to `can-map`.